src/popup/qrcode.html

```diff
@@ -33,15 +33,15 @@
 					data-i18n
 					data-i18n-alt="__MSG_qrCodePlaceholder__">
 				<div class="message-container center-container">
-					<div id="messageInfo" class="message-box info flot-qr float-qr-bottom invisible fade-hide">
+					<div id="messageInfo" aria-label="info" class="message-box info flot-qr float-qr-bottom invisible fade-hide">
 						<span class="message-text"></span>
 						<img class="icon-dismiss invisible" src="/common/img/close.svg" width="24" height="24" tabindex="0" data-i18n data-i18n-aria-label="__MSG_dismissIconDescription__"></span>
 					</div>
-					<div id="messageError" class="message-box error flot-qr float-qr-bottom invisible fade-hide">
+					<div id="messageError" aria-label="error" class="message-box error flot-qr float-qr-bottom invisible fade-hide">
 						<span class="message-text" data-i18n="__MSG_couldNotGenerateQrCode__">Could not generate QR code.</span>
 						<img class="icon-dismiss invisible" src="/common/img/close-white.svg" width="24" height="24" tabindex="0" data-i18n data-i18n-aria-label="__MSG_dismissIconDescription__"></span>
 					</div>
-					<div id="messageLoading" class="message-box info flot-qr float-qr-bottom">
+					<div id="messageLoading" aria-label="info" class="message-box info flot-qr float-qr-bottom">
```

Comment:
```suggestion
					<div id="messageLoading" aria-label="loading" class="message-box info flot-qr float-qr-bottom">
```

The visual style is "info", but I would be fine with proving more info here, so "info box: loading" (or the thing I've adjusted above) may be an idea.